### PR TITLE
Feature shp

### PR DESCRIPTION
**Pull request by phong1233**

### CODE REVIEW

File: src/main/java/org/team4/common/SimulationClock.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+        Date curr = (Date) argCurr.clone();
+        Date on  = (Date) argCurr.clone();
+        on.setHours(argOn.getHours());
+        on.setMinutes(argOn.getMinutes());
+        on.setSeconds(argOn.getSeconds());
+        Date off  = (Date) argCurr.clone();
+        off.setHours(argOff.getHours());
+        off.setMinutes(argOff.getMinutes());
+        off.setSeconds(argOff.getSeconds());
```
Comment (Sebastien0730) [inline src/main/java/org/team4/common/SimulationClock.java:135]: Are these methods deprecated?

File: src/main/java/org/team4/common/SimulationClock.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        Date off = Settings.awayLightOffTime;
+
+        Boolean targetInZone = isBettweenTime(curr, on, off);
+
+        if(targetInZone && !Settings.awayLightOn) {
+            House.turnOnAllAwayModeLights();
+            Logger.info("All away mode lights have been turned on");
+            Settings.awayLightOn = true;
+            DashboardController dashboardController = App.fxmlLoader.getController();
+            dashboardController.drawHouseLayout();
```
Comment (Sebastien0730) [inline src/main/java/org/team4/common/SimulationClock.java:70]: This should be done using an observer or something?

File: src/main/java/org/team4/common/SimulationClock.java (hunk 1 above)
Comment (phong1233) [reply src/main/java/org/team4/common/SimulationClock.java:135]: Old but gold

File: src/main/java/org/team4/common/SimulationClock.java (hunk 2 above)
Comment (phong1233) [reply src/main/java/org/team4/common/SimulationClock.java:70]: Too much effort

Comment (Sebastien0730) [reply src/main/java/org/team4/common/SimulationClock.java:70]: True

**Review by Sebastien0730 — APPROVED** [review]
Looking good team 3

**Review by successsb89 — APPROVED** [review]

**Pull request merged**